spec/support/user_helper.rb

```diff
@@ -0,0 +1,7 @@
+def login(email)
+  visit root_path
+  click_link 'Log In'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.